python/custreamz/README.md

```diff
@@ -0,0 +1,4 @@
+1. cuStreamz is a GPU-accelerated Streaming Library, which uses cuDF with Streamz for stream data processing on GPUs. 
+2. cuStreamz has its own conda metapackage which makes it as simple as possible to install the set of dependencies necessary to process streaming workloads on GPUs.
+3. A series of tests for use in a cuDF gpuCI instance have been included ensuring that changes continuously rolled out as part of cuDF don't break its integration with Streamz.
+4. You can find example notebooks on how to write cuStreamz jobs in the RAPIDS notebooks-extended repository.  
```

Comment:
```suggestion
4. You can find [example](https://github.com/rapidsai/notebooks-contrib/blob/master/getting_started_notebooks/basics/hello_streamz.ipynb) [notebooks](https://github.com/rapidsai/notebooks-contrib/blob/master/getting_started_notebooks/basics/streamz_weblogs.ipynb) on how to write cuStreamz jobs in the RAPIDS [notebooks-contrib repository](https://github.com/rapidsai/notebooks-contrib).  
```

Reply:
Had thought of doing this! Should we not add a link to the directory (hopefully going to be added with the custreamz PR against notebooks-extended) with the streaming word count & HAProxy examples instead?

Reply:
Shoot!  I re-requested a review, since I hadn't seen this before. Sorry about that! :(

Reply:
@chinmaychandak just waiting on this change before we can hand it over to ops, let me know when this is done.

Reply:
>Should we not add a link to the directory (hopefully going to be added with the custreamz PR against notebooks-extended) with the streaming word count & HAProxy examples instead?
@randerzander

Reply:
let's keep that a separate discussion